### PR TITLE
Unify CSS classes across editor UI forms.

### PR DESCRIPTION
**Pull request by pszczesniak**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Other (bookmark): Form styles and structure is now unified with use of `ck-form` and `ck-form__row`.  

Other (image): Image custom resize form styles and structure now unified with use of `ck-form` and `ck-form__row`.  

Other (link): Form styles and structure is now unified with use of `ck-form` and `ck-form__row`.  

Other (table): Extracted the `form.css` to `theme-lark` package.  

Other (table): Extracted the `FormRowView` to a separate package.  

Feature (theme-lark): Introduced the `form.css` component .  

Feature (ui): Introduced the `FormRowView` class.

MINOR BREAKING CHANGE (table): The `FormRowView` class was moved to the `ui` package.  

MINOR BREAKING CHANGE (table): The `form.css` component was moved to the `theme-lark` package.

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

File: packages/ckeditor5-ui/theme/components/form/form.css [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+			padding-bottom: var(--ck-spacing-large);
+		}
+
+		&.ck-form__row_with-submit {
+			flex-wrap: nowrap;
+
+			& > *:not(:first-child) {
+				margin-inline-start: var(--ck-spacing-standard);
+			}
+		}
```
Comment (pszczesniak) [inline packages/ckeditor5-ui/theme/components/form/form.css:39]: Move those properties to the `ckeditor5-ui/theme/components/formrow/formrow.css`.

Comment (pszczesniak) [reply packages/ckeditor5-ui/theme/components/form/form.css]: Addressed in c488617.

**Review by Mati365 — APPROVED** [review]
I tested, it works fine. It looks like there is no visual difference between old version and current one (at least no big one) and the code is much less redundant :tada:

**Comment by niegowski** [comment]
The "Suggested merge commit message" should be separated with paragraphs, not BRs.

**Review by niegowski — APPROVED** [review]

**Pull request merged**